deploy.sh

```diff
@@ -54,14 +54,19 @@ fi
 SVN_URL="https://plugins.svn.wordpress.org/${SLUG}/"
 SVN_DIR="${HOME}/svn-${SLUG}"
 
-# Checkout just trunk and assets for efficiency
-# Tagging will be handled on the SVN level
+# Checkout SVN repository.
 echo "➤ Checking out .org repository..."
 svn checkout --depth immediates "$SVN_URL" "$SVN_DIR"
 cd "$SVN_DIR"
 svn update --set-depth infinity assets
 svn update --set-depth infinity trunk
+svn update --set-depth infinity tags
```

Comment:
Since we only need to confirm the directory exists, and not get all the files, we should be able to make use of the `immediates` argument for sparse checkouts, which will be a bit faster. like this:

```suggestion
svn update --set-depth immediates tags
```

That will result in getting empty directories for all of the tags that have been published. 
See: https://svnbook.red-bean.com/en/1.6/svn.advanced.sparsedirs.html

Alternately, we could just try to check out the specific tag being passed as the `$VERSION` variable, but the above example should probably work fine. Something like:

```
svn update --set-depth infinity "tags/$VERSION"
```

However, the error handling would be a bit more difficult to deal with so the previous is probably sufficient.

Reply:
Thanks @joemcgill, I tested the suggestion, and it worked fine.